snow/consensus/snowman/topological.go

```diff
@@ -289,7 +289,7 @@ func (ts *Topological) RecordPoll(ctx context.Context, voteBag bag.Bag[ids.ID])
 
 	// Runtime = |live set| ; Space = Constant
 	// Traverse from the preferred ID to the last accepted ancestor.
-	for block := startBlock; !block.Accepted(); {
+	for block := startBlock; !block.Decided(); {
```

Comment:
Could you add a comment for future reference why it's alright to switch from `Accepted()` to `Decided()` here?

Reply:
My understanding: this code iterates from the preference to the last accepted ancestor, which guarantees that the blocks can only be Processing or Accepted (when it hits the last accepted block). It should never hit a rejected block, which makes this change a no-op.

Reply:
added a comment